react-native/src/blueTheme.ts

```diff
@@ -6,83 +6,53 @@ All rights reserved.
 This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
 **/
 
-import { blue, red, gray, lightBlue, white } from '@pxblue/colors';
-import { TextStyle } from 'react-native';
+import { Platform } from 'react-native';
+import { configureFonts, DefaultTheme, Theme } from 'react-native-paper';
+import { blue, red, gray, black, lightBlue, white } from '@pxblue/colors';
 
-type Font = {
-    fontFamily: string;
-    fontWeight: TextStyle['fontWeight'];
+const fallbackFont = Platform.OS === 'ios' ? 'System' : 'Roboto';
+type FontWeight = 'normal' | 'bold' | '100' | '200' | '300' | '400' | '500' | '600' | '700' | '800' | '900';
+
+const defaultFontConfig = {
+    regular: {
+        fontFamily: 'OpenSans-Regular' || fallbackFont,
+        fontWeight: '400' as FontWeight,
+    },
+    medium: {
+        fontFamily: 'OpenSans-SemiBold' || fallbackFont,
+        fontWeight: '500' as FontWeight,
```

Comment:
for open sans this has to be 600 to make it appear semi bold

Reply:
The fontWeight here actually doesn't have any impact. The weight comes entirely from the fontFamily. For some reason RN has trouble with using the font Family name (Open Sans) and picking up the weights as you'd expect. That's why we had to specify the font family individually for each weight. Kind of annoying. I'll update the weight to 600 to be correct anyway.